bin/i18n/hoc_sync_utils.rb

```diff
@@ -16,7 +16,7 @@ def self.sync_in
     dest_dir = File.join(I18N_SOURCE_DIR, "hourofcode")
 
     # Copy the file containing developer-added strings
-    Dir.mkdir(dest_dir) unless Dir.exist?(dest_dir)
+    FileUtils.mkdir_p(dest_dir)
```

Comment:
Sorry could you remove this? It is just a general sync script fix. I will do that in a separate PR.